metadata.yml

```diff
@@ -6,6 +6,9 @@ rules:
 mutating: false
 contextAware: false
 executionMode: kubewarden-wapc
+# Flag to allow the audit scanner to send request to this policy while scanning
+# resources running in the cluster. The default value of this field is "true"
```

Comment:
I think it would be better to be more end-user focused, and also mention the intrinsic limitations of background audit feature.
Something like:
```
Consider the policy for the background audit scans. Default is true. Note the intrinsic limitations of the background audit feature on docs.kubewarden.io; If your policy hits any limitations, set to false for the audit feature to skip this policy and not generate false positives.
```